pkg/client/client.go

```diff
@@ -178,6 +179,10 @@ func NewClientFromKeys(accessKey, secretAccessKey, region string) (Client, error
 	if err != nil {
 		return nil, err
 	}
+	s.Handlers.Build.PushBackNamed(request.NamedHandler{
+		Name: "sigs.k8s.io.ClusterAPIProviderAWS",
+		Fn:   request.MakeAddToUserAgentHandler("sigs.k8s.io ClusterAPIProviderAWS", "dummy"),
```

Comment:
@enxebre do we want more fancy name?

@abhinavdahiya I would rather go with `openshift.io` rather than `sigs.k8s.io`. Can you go with `openshift.io cluster-api-provider-aws`? The first part describing the group, the second part repository of origin.

Reply:
done with https://github.com/openshift/cluster-api-provider-aws/compare/815c929f51df180fcf72af0a0feb6156a893fb6f..8338c4fb0b0ad3b2e62af0fb203173ca4bd36f39